src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -94,6 +99,21 @@ endif()
 if (NOT OpenMVG_USE_INTERNAL_CEREAL)
   find_dependency(cereal REQUIRED)
 endif()
+if (NOT OpenMVG_USE_INTERNAL_CERES)
+  find_dependency(Ceres REQUIRED)
+endif()
+if (NOT OpenMVG_USE_INTERNAL_COINUTILS)
+  find_dependency(CoinUtils REQUIRED)
+endif()
+if (NOT OpenMVG_USE_INTERNAL_CLP)
+  find_dependency(Clp REQUIRED)
+endif()
+if (NOT OpenMVG_USE_INTERNAL_OSI)
+  find_dependency(Osi REQUIRED)
+endif()
+if (NOT OpenMVG_USE_INTERNAL_LEMON)
+  find_dependency(Lemon REQUIRED)
+endif()
```

Comment:
It seems like all of these are required if not using the internal version, but this can be changed if that's not actually so.

Reply:
Yes they are most likely required, if they are listed as PUBLIC dependency or listed in some header files.